### PR TITLE
add syntax highlighting of javadoc

### PR DESCRIPTION
**Pull request by qsdrqs**
I add some scripts to enable syntax highlighting of javadoc.
But one thing no great is that I need vim's default `html.vim` to help me highlighting html but I don't know how read default `html.vim'. So I copied it into plug directory.

### CODE REVIEW

**Comment by uiiaoo** [comment]
Thank you for your contribution.
Please give me some time until the reply.

File: syntax_body/java/syntax/pattern.vim [hunk 1]
```diff
@@ -31,3 +31,23 @@ sy match  javaInclude       '\v<import%(\_s+static)=>'
 sy match  javaPackagePath   '\v<%(%(\w|\$)+\_s*\.\_s*)*%(\w|\$)+>'
 \   contained contains=javaIdentifier,javaOperator
 "---------------------------------------------------------------------------------------------------
+if !exists("java_ignore_javadoc") && main_syntax != 'jsp'
+  " syntax coloring for javadoc comments (HTML)
+  syntax include @javaHtml <sfile>:p:h/html.vim
```
Comment (uiiaoo) [inline syntax_body/java/syntax/pattern.vim]: This command can be replaced by the following command without the default `html.vim`.
```suggestion
  syntax include @javaHtml syntax/html.vim
```

File: syntax_body/java/hilink/javadoc.vim [hunk 2]
```diff
@@ -0,0 +1,8 @@
+hi def link javaDocTags     Special
+hi def link javaDocParam        Function
+hi def link javaDocSeeTagParam      Function
+hi def link javaDocComment      Comment
+hi def link javaCommentTitle		SpecialComment
+
+hi def link htmlComment		Special
+hi def link htmlCommentPart		Special
```
Comment (uiiaoo) [inline syntax_body/java/hilink/javadoc.vim]: I don't like that spaces and tabs are used together.
So I would like you to use only spaces like other files on this repository.

**Comment by uiiaoo** [comment]
The code you brought in is a part of the default `java.vim`.
Doesn't it violate Vim's license? Because MIT license is applied to this repository.

File: syntax_body/java/hilink/javadoc.vim (hunk 2 above)
Comment (qsdrqs) [reply syntax_body/java/hilink/javadoc.vim]: Ok! I have format it as other files in this repository. Please have a look and point out is there other things that I need to do? Thank you.

**Comment by qsdrqs** [comment]
Actually I don't know much about license conflicts. But in my opinion the MIT license should be compatible with vim license because they both compatible with GPL license. For example neovim use Apache license. So I don't think it violate vim's license.

**Comment by uiiaoo** [comment]
I apologize for not replying to you for a long time.
Neovim is avoiding this problem by using both of the licenses together.
I don't like this method for this repository, so I can't accept your commits.
As for the highlighting of Javadoc, I will consider introducing it.
Thank you.

**Pull request closed without merging**